### PR TITLE
Added Simple Gradient Descent example for Tensorflow.

### PR DESCRIPTION
**Pull request by yashbmewada**

### CODE REVIEW

**Comment by rathodchintan** [comment]
@yashbmewada This is very helpful example. Thanks for adding.

**Comment by yashbmewada** [comment]
Thank you for accepting the pull request. @rathodchintan would be adding more soon.
